test/intl402/NumberFormat/prototype/format/value-tonumber.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2018 Ujjwal Sharma. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-number-format-functions
+description: >
+  Tests that Intl.NumberFormat.prototype.format converts its argument
+  (called value) to a number using ToNumber (7.1.3).
+info: |
+  11.1.4Number Format Functions
+
+  4. Let x be ? ToNumber(value).
+features: [Symbol]
+---*/
```

Comment:
Ideally you'd want a test that patches @@toPrimitive to make sure that the ToNumber is actually being called